orb-ui/src/engine/center/alert.rs

```diff
@@ -24,10 +32,16 @@ pub struct Alert<const N: usize> {
 impl<const N: usize> Alert<N> {
     /// Creates a new [`Alert`].
     #[must_use]
-    pub fn new(color: Argb, pattern: Vec<f64>, start_on: bool) -> Self {
+    pub fn new(
+        color: Argb,
+        pattern: Vec<f64>,
+        smooth_transitions: Option<Vec<f64>>,
+        start_on: bool,
```

Comment:
Can you rename to "leds_active_at_start" or something like that so that its more clear what this is?